### PR TITLE
Remove beta from default storage class

### PR DESCRIPTION
**Pull request by pipo02mix**
Current Kubernetes version complaints about `default` storage because of having an old annotation parameter.

### CODE REVIEW

File: service/controller/clusterapi/v27/templates/cloudconfig/instance_storage_class.go [hunk 1]
```diff
@@ -1,11 +1,11 @@
 package cloudconfig
 
-const InstanceStorageClassContent = `apiVersion: storage.k8s.io/v1beta1
+const InstanceStorageClassContent = `apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: gp2
   annotations:
-    storageclass.beta.kubernetes.io/is-default-class: "true"
+    storageclass.kubernetes.io/is-default-class: "true"
```
Comment (xh3b4sd) [inline service/controller/clusterapi/v27/templates/cloudconfig/instance_storage_class.go]: You probably want to do this for the legacy controller resources as well. Maybe also other versions if necessary. 

```
service/controller/legacy/v27/templates/cloudconfig/instance_storage_class.go
```

Comment (pipo02mix) [reply service/controller/clusterapi/v27/templates/cloudconfig/instance_storage_class.go]: but can I modify if they are not WIP? or it `clusterapi` no wired yet?

Comment (xh3b4sd) [reply service/controller/clusterapi/v27/templates/cloudconfig/instance_storage_class.go]: v27 is WIP. And clusterapi is not wired.

Comment (pipo02mix) [reply service/controller/clusterapi/v27/templates/cloudconfig/instance_storage_class.go]: done

**Pull request merged**